### DISCUSSION TITLE
Why using uwsm/systemd is better than just launching hyprland from tty?

### DISCUSSION THREAD

**Discussion by azizsayfiddin** [hyprwm/Hyprland#8817] [Q&A]
I noticed that when I launch Hyprland with uwsm, it consumes more RAM than if I would launch it directly from TTY. What benefits are there for using Hyprland with uwsm/systemd?

**Comment by fufexan** [comment]
More RAM? Can you give us some numbers?

Benefits are mentioned in the wiki and detailed in [uwsm's readme](https://github.com/Vladimir-csp/uwsm?tab=readme-ov-file#universal-wayland-session-manager).

**Reply by azizsayfiddin** [reply]
When I launch from uwsm, it uses ~1.5GGiB, when just with TTY 1 GiB

**Comment by izmyname** [comment]
Link my reply to the issue with the same name https://github.com/hyprwm/Hyprland/issues/8818#issuecomment-2559316248

**Reply by azizsayfiddin** [reply]
Did I create issue? I don't remember that I created it.
